firebase-appdistribution/src/main/res/layout/activity_feedback.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file generated somehow?

Reply:
No I built this. It's a very basic prototype UI. You can see it in the screencast in the PR description.